### PR TITLE
fix docs home page on mobile renders poorly

### PR DESCRIPTION
**Pull request by Arhell**
Before:
a37d9a6435
After:

#19671

### CODE REVIEW

**Comment by sftim** [comment]
Hi @Arhell 

To review this, I'll want to check that the code changes match the PR description. Could you summarize what changes this PR makes and (briefly) why those are useful?

Thank you for the screen grabs.
If you can provide before and after links, that will make reviewing even easier.

**Comment by Arhell** [comment]
@sftim sure
check on mobile 320+
Before: https://kubernetes.io/docs/home/
After: https://deploy-preview-19686--kubernetes-io-master-staging.netlify.com/docs/home/ this PR

File: static/css/gridpage.css [hunk 1]
```diff
@@ -17,12 +17,18 @@
...
-    width: 33%;
-    padding: 0 30px 0 0;
-    margin-bottom: 50px;
+    margin-bottom: 30px;
     min-height: 152px;
 }
 
+@media (min-width: 768px){
+    .launch-card {
+        width: 33%;
```
Comment (kbhawkey) [inline static/css/gridpage.css:26]: Hello @Arhell .
I compared the changes to the current version of the docs home page (minimized desktop browser).
Here is what looks different with the changes:
- I see a vertical scroll bar in the vendorStrip
- An empty flyout-button appears on the left top corner (normally invisible)
- Text sections now left aligned.

Comment (Arhell) [reply static/css/gridpage.css:26]: @kbhawkey 
Hi, of course, because the wizard already has more recent changes, need to do a rebase with master

Comment (Arhell) [reply static/css/gridpage.css:26]: Please look only at these blocks.
As you can see by the code, they only changed

File: static/css/gridpage.css [hunk 2]
```diff
@@ -17,12 +17,18 @@
...
 .launch-card {
     position: relative;
-    width: 33%;
-    padding: 0 30px 0 0;
-    margin-bottom: 50px;
+    margin-bottom: 30px;
     min-height: 152px;
 }
 
+@media (min-width: 768px){
```
Comment (sftim) [inline static/css/gridpage.css:24]: Looking at @kbhawkey's feedback, I suggest:
```suggestion
@media screen and (max-width: 768px){
```
?

**Comment by Arhell** [comment]
@sftim @kbhawkey done & rebase

File: static/css/gridpage.css [hunk 3]
```diff
@@ -23,6 +23,15 @@
     min-height: 152px;
 }
 
+@media screen and (max-width: 768px){
+    .launch-card {
+        width: 100%;
```
Comment (kbhawkey) [inline static/css/gridpage.css:28]: :wave: @Arhell .
What do you think about grouping this addition with the existing `@media screen ` entries (which start at line 254)?

Comment (Arhell) [reply static/css/gridpage.css]: @kbhawkey  hi!
excellent thank you!
Moved

**Comment by kbhawkey** [comment]
@sftim, Do you have additional comments?
